cumulusci/core/utils.py

```diff
@@ -116,10 +121,18 @@ def process_list_of_pairs_dict_arg(arg):
         rc = {}
         for key_value in arg.split(","):
             subparts = key_value.split(":")
-            if len(subparts) == 2:
+            # Support URI/IRI in the argument
+            match = URI_RE.search(key_value)
+            if subparts[0] in rc:
+                raise TaskOptionsError(f"Var specified twice: {subparts[0]}")
+            elif match:
+                uri = match.group()
+                if "http" in subparts[0]:
+                    raise TaskOptionsError(f"No name given for URI: {uri}")
+                key = subparts[0]
+                rc[key] = uri
```

Comment:
Like I mentioned in slack, let's avoid using a regex to match URIs, and simply do key_value.split(":", 1) on line 123 to make sure we only split on the first colon.